src/main/java/com/mael/ttt/players/RobotPlayer.java

```diff
@@ -72,7 +73,7 @@ private boolean gameIsNotOver(String currentPlayer) {
     }
 
     private String swapMark(String currentPlayer) {
-        return (currentPlayer.equals("X") ? "O" : "X");
+        return (currentPlayer.equals(Marks.PLAYER.toString()) ? Marks.OPPONENT.toString() : Marks.PLAYER.toString());
```

Comment:
Why are you not just comparing the `Marks`? Do you have to compare `String` objects here?